src/structures/Guild.js

```diff
@@ -328,6 +335,10 @@ class Guild extends Base {
       this.approximatePresenceCount = null;
     }
 
+    if (typeof data.welcome_screen !== 'undefined') {
```

Comment:
```suggestion
    if (data.welcome_screen) {
```

Reply:
would be inconsistent with the properties above, as the property might not be populated, although present further down the line if discord actually decides to provide updates to these in guild payloads